components/list/list-item-drag-drop-mixin.js

```diff
@@ -427,13 +546,32 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 			e.dataTransfer.setData('text/plain', `${this.dropText}`);
 		}
 
-		// Legacy-Edge doesn't support setDragImage. Experience is not degraded for Legacy-Edge by doing this fix.
-		if (e.dataTransfer.setDragImage) {
-			const nodeImage = this.shadowRoot.querySelector('.d2l-list-item-drag-image') || this;
-			e.dataTransfer.setDragImage(nodeImage, 50, 50);
-		}
+		const nodeImage = this.shadowRoot.querySelector('.d2l-list-item-drag-image') || this;
+		e.dataTransfer.setDragImage(nodeImage, 50, 50);
+
+		const rootList = this._getRootList(this);
 
-		createDragState(this);
+		// getSelectionInfo(false) is fast so we can quickly check the state
+		if (!rootList.dragMultiple || rootList.getSelectionInfo(false).state === SelectionInfo.states.none) {
+			createDragState([this]);
+		} else {
+
+			// get the seelcted items, but do not include selected items of selected items
+			const getDragTargets = list => {
+				let dragTargets = [];
+				list.getItems().forEach(item => {
```

Comment:
In this method, we preserve the order to the items being dragged.  Also worth mentioning, we do not list nested items of items being dragged, as it is already implied that their subtree is being dragged intact (i.e. we do not want the consumer to flatten it).